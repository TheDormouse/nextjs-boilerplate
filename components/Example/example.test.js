import ReactDOM from 'react-dom';
import { Example } from './';

// A straightforward link wrapper that renders an <a> with the passed props. What we are testing
// here is that the Link component passes the right props to the wrapper and itself.
const LinkWrapper = props => <a {...props} />; // eslint-disable-line jsx-a11y/anchor-has-content

it('Renders "Hello World"', () => {
  const div = document.createElement('div');
  ReactDOM.render(
   <Example />,
    div
  );

  expect(div.textContent).toEqual('Hello World');

  ReactDOM.unmountComponentAtNode(div);
});
