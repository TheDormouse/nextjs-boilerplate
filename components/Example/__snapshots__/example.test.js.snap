// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "Hello World" 1`] = `
<div>
  <div
    class="example "
  >
    Hello World
  </div>
</div>
`;
